packages/blade/src/components/Tabs/_decisions/decisions.md

```diff
@@ -0,0 +1,385 @@
+   * Can be used to render an badge/counter.
+   */
+  trailing?: React.ReactNode;
+  /**
+   * If `true`, the tab item will be disabled.
+   */
+  isDisabled?: boolean;
+};
+```
+
+## `TabPanel` API
+
+```ts
+type TabPanelProps = {
+  /**
+   * The value of the tab panel. This will be used to match the selected tab.
+   */
+  value: string;
+  /**
+   * The content of the tab panel.
+   */
+  children: React.ReactNode;
+};
+```
+
+### Examples:
+
+## Basic
+
+```js
+<Tabs>
+  <TabList>
+    <TabItem value="payments">Payments</TabItem>
+    <TabItem value="refunds">Refunds</TabItem>
+    <TabItem value="disputes">Disputes</TabItem>
+    <TabItem value="settlements">Settlements</TabItem>
+  </TabList>
+
+  <TabPanel value="payments">
+    <Payments />
+  </TabPanel>
+  <TabPanel value="refunds">
+    <Refunds />
+  </TabPanel>
+  <TabPanel value="disputes">
+    <Disputes />
+  </TabPanel>
+  <TabPanel value="settlements">
+    <Settlements />
+  </TabPanel>
+</Tabs>
+```
+
+<img src="./example-1.png" width="380" alt="Tabs Example 1" />
+
+#### Controlled, Uncontrolled
+
+```js
+const Controlled = () => {
+  const [selected, setSelected] = React.useState('payments');
+
+  return (
+    <Tabs value={selected} onChange={(value) => setSelected(value)}>
+      <TabList>
+        <TabItem value="payments">Payments</TabItem>
+        <TabItem value="refunds">Refunds</TabItem>
+      </TabList>
+
+      <TabPanel value="payments">
+        <Payments />
+      </TabPanel>
+      <TabPanel value="refunds">
+        <Refunds />
+      </TabPanel>
+    </Tabs>
+  );
+};
+
+const Uncontrolled = () => {
+  return (
+    <Tabs defaultValue="refunds">
+      <TabList>
+        <TabItem value="payments">Payments</TabItem>
+        <TabItem value="refunds">Refunds</TabItem>
+      </TabList>
+
+      <TabPanel value="payments">
+        <Payments />
+      </TabPanel>
+      <TabPanel value="refunds">
+        <Refunds />
+      </TabPanel>
+    </Tabs>
+  );
+};
+```
+
+## Vertical Orientation
+
+```jsx
+<Tabs orientation="vertical">
+  <TabList>
+    <TabItem value="all-expenses" leading={RupeeIcon}>
+      All Expenses
+    </TabItem>
+    <TabItem
+      value="categories"
+      leading={EditIcon}
+      trailing={<Counter value={26} variant="notice" />}
+    >
+      Categorise
+    </TabItem>
+    <TabItem
+      value="review"
+      leading={TransactionIcon}
+      trailing={<Counter value={99} variant="information" />}
+    >
+      Review
+    </TabItem>
+    <TabItem value="synced" leading={CheckCircleIcon}>
+      Synced
+    </TabItem>
+    <TabItem
+      value="synced-fail"
+      leading={WarningIcon}
+      trailing={<Counter value={3} variant="negative" />}
+    >
+      Synced Failed
+    </TabItem>
+    <TabItem value="excluded" leading={SlashIcon}>
+      Excluded
+    </TabItem>
+  </TabList>
+  ...
+</Tabs>
+```
+
+<img src="./example-4.png" width="380" alt="Tabs Example 2" />
+
+## Filled Variant
+
+```jsx
+<Tabs defaultValue="refunds">
```

Comment:
update this. typo here. it doesn't has filled variant and the default value is something else

Reply:
Updated.